dolphinscheduler-ui/src/common/common.ts

```diff
@@ -49,6 +50,31 @@ export const bytesToSize = (bytes: number) => {
   return parseFloat((bytes / Math.pow(k, i)).toPrecision(3)) + ' ' + sizes[i]
 }
 
+export function renderEnvironmentalDistinctionCell(
+  testFlag: number | undefined,
+  t: Function
+) {
+  if (testFlag === 0) {
+    return h(
+      NTag,
+      { type: 'success', size: 'small' },
+      {
+        default: () => t('datasource.on_line')
+      }
+    )
+  } else if (testFlag === 1) {
+    return h(
+      NTag,
+      { type: 'warning', size: 'small' },
+      {
+        default: () => t('datasource.test')
+      }
+    )
+  } else {
+    return '-'
+  }
+}
+
```

Comment:
The common folder is used for configuration-related information. It's better to move it to the util folder by a separate file.